### ISSUE TITLE
Test runner can't find my tests

### ISSUE THREAD

**Issue by robfreundlich** [mbolotov/intellij-cypress#88]
Using the basic or pro versions, I can't get the plugin to find my tests. Here's my directory structure, the location and contents of my `cypress.json` file, and the location of my spec files:

```
/c/dev/mittrock-spotify$ tree -d -I node_modules
.
├── cypress
│   ├── downloads
│   ├── fixtures
│   ├── integration
│   │   └── client
│   │       ├── model
│   │       └── utils
│   ├── plugins
│   └── support
├── dist
├── docs
└── src
    └── client
        ├── app
        ├── browser
        ├── model
        ├── player
        └── utils

18 directories
/c/dev/mittrock-spotify$ find . -name cypress.json
./cypress.json
/c/dev/mittrock-spotify$ cat cypress.json
{
  "integrationFolder": "cypress/integration",
  "fixturesFolder": "cypress/fixtures",
  "testFiles": "**/*.spec.ts"
}
/c/dev/mittrock-spotify$ find . -name "*.spec.ts"
./cypress/integration/client/model/Genre.spec.ts
./cypress/integration/client/model/Title.spec.ts
./cypress/integration/client/utils/ArrayUtils.spec.ts
/c/dev/mittrock-spotify$
```

When I try to run all tests in the `cypress/integration` directory, I get this:
![image](https://user-images.githubusercontent.com/7975013/164990901-248b4965-6d32-4bad-b850-0078a614c100.png)

When I try to run a specific test file, I get this:
![image](https://user-images.githubusercontent.com/7975013/164990931-e83013cf-4ef0-425f-ba71-40ca3de6fbf7.png)

I can run from the command line just fine:
![image](https://user-images.githubusercontent.com/7975013/164991022-ead12ebf-e8b1-45c8-b36b-ca25e2b356f0.png)

Here's the "About IntelliJ" information:
```
IntelliJ IDEA 2021.3.3 (Ultimate Edition)
Build #IU-213.7172.25, built on March 15, 2022
Licensed to Rob Freundlich
You have a perpetual fallback license for this version.
Subscription is active until April 8, 2023.
Runtime version: 11.0.14.1+1-b1751.46 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Windows 10 10.0
GC: G1 Young Generation, G1 Old Generation
Memory: 2030M
Cores: 8
Non-Bundled Plugins:
    com.tabnine.TabNine (0.6.0)
    com.github.b3er.idea.plugins.arc.browser (0.32)
    de.nordgedanken.auto_hotkey (0.10.0)
    HighlightBracketPair (1.3.4)
    mobi.hsz.idea.gitignore (4.4.0)
    com.mallowigi.colorHighlighter (9.2.0)
    me.mbolotov.cypress.pro (3.0)

Kotlin: 213-1.5.10-release-949-IJ7172.25
```

**Comment by mbolotov** [comment]
Hi Rob,
That is a known problem. Actually, this is a false alarm and you can simply click on 'Run' (and probably on another one) to start a test. 
I'm planning to release a new version with this fix later this month.
Meanwhile, here is a bundle with fix you can download and [install](https://www.jetbrains.com/help/idea/managing-plugins.html#install_plugin_from_disk):
[Cypress Support Pro-3.1-EAP.zip](https://github.com/mbolotov/intellij-cypress/files/8550393/Cypress.Support.Pro-3.1-EAP.zip)

**Comment by robfreundlich** [comment]
That solved it. Thanks!

**Comment by mbolotov** [comment]
let's keep this issue open until the release

**Comment by mbolotov** [comment]
this should be fixed in 3.1 version
